### PR TITLE
Reskin Hind

### PR DESCRIPTION
**Pull request by PunkPun**
<details><summary>**Hind reskinned and renamed to Black Hawk**</summary>

bhcolor

bwbh</details>

This is also a long requested and controversial feature as Hind is a legendary Soviet aircraft has no place in the hands of Allies.

**Notes:** 
- Sprite and Icon were made by Inq
- Black Hawk is not the final name and is still being discussed
- File names are to be decided

### CODE REVIEW

File: mods/ra/rules/aircraft.yaml
Comment (MustaphaTR) [inline mods/ra/rules/aircraft.yaml]: There is a tabbing error here, also please use 40 and move everything else up instead of using 35.

File: mods/ra/rules/aircraft.yaml
Comment (MustaphaTR) [inline mods/ra/rules/aircraft.yaml]: Second sentence shouldn't start with a space, also either there should be a dot after "helicopter" or "Armed" should be "armed". The dot after "Infantry" should go back to after "chainguns".

File: mods/ra/rules/aircraft.yaml
Comment (MustaphaTR) [inline mods/ra/rules/aircraft.yaml]: More spaces instead of tabs.

File: mods/ra/rules/aircraft.yaml
Comment (MustaphaTR) [inline mods/ra/rules/aircraft.yaml]: Ditto.

File: mods/ra/rules/aircraft.yaml
Comment (MustaphaTR) [inline mods/ra/rules/aircraft.yaml]: You get the point, there is here and some below and almost all of the weapon and sequence changes use spaces instead of tabs.

**Review by MustaphaTR — DISMISSED** [review]

File: mods/ra/rules/aircraft.yaml
Comment (abcdefg30) [reply mods/ra/rules/aircraft.yaml]: To be fair: IIRC the original motivation of having the build order as multiples of 10 was exactly to allow inserting other actors in-between without having to adjust all following actors.

Comment (MustaphaTR) [reply mods/ra/rules/aircraft.yaml]: I think the values in main mods should stay as multipiles of 10, but it helps with the custom maps, which i think should be the reasoning for it.

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: It seems that some of the copy-pasted text had spaces or that they were created when pasting

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: fixed

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: Moved Longbow and Mig up by 10 to fit Hind in 40

**Comment by GraionDilach** [comment]
How does this affects the campaign? Repurposing the Hind to the transport from it's attack copter role might affect mission performance. Note that in legacy RA1 the Chinook was shared between both factions, so this pretty much shifts the Hind to take over both helis' place in the campaigns.

TBH, this already seems to be still a more promising start in all fronts than #12836 & related though.

File: mods/ra/sequences/aircraft.yaml
Comment (abcdefg30) [inline mods/ra/sequences/aircraft.yaml]: Please revert this change.

**Comment by matjaeck** [comment]
> How does this affects the campaign? Repurposing the Hind to the transport from it's attack copter role might affect mission performance. Note that in legacy RA1 the Chinook was shared between both factions, so this pretty much shifts the Hind to take over both helis' place in missions.

I personally think we shouldn't transform the hind in a transport helicopter. One reason is that the sprite is so thin and small and it just does not look right to me. The other point is that we can't give soviets air transport as a passive ability of an already powerful weapon. You should have to invest into (mass) air transport. If soviets really need a transport heli, then let's give them the Chinook, but leave the hind in original state. We diverge even further from it with the proposed setup.

Another thing I disagree with is that helicopters are build from airfields: IMO it's a good thing if players have to decide between helicopters and planes - it will allow more variety in build orders and strategy and the helipad and its loading animation with the landed hind is something I would *really* miss.

I'm not convinced of the Blackhawk helicopter in its role as replacement for the hind because it is quite big and looks like it would be able to transport something - it is not an attack helicopter. It does not look right, although the artwork does look good. What might work is to use it as an armed support helicopter that can load some infantry. Then allies don't need the Chinook anymore and we can give it to Soviets, Hind is restored to its owner and allies get a new toy.

I stay skeptical because of these points and in general because it is such an high impact change. I would have hoped someone with more experience of the project and community had taken the lead here.

**Comment by Orb370** [comment]
From a balance/gameplay perspective I think this needs more testing.  I haven't seen too much wide scale testing, and I think these kind of changes need to be thoroughly vetted.

The few games I did test I didn't see the unit used, except my for my few failed attempts to abuse it.

Also, I recently realized it's a bit strange that it fires so slowly. I get this is how the Blackhawk works in RA2, but it still doesn't make sense. A fast rate of fire weapon with low damage per hit would be better.

File: mods/ra/sequences/aircraft.yaml
Comment (Punsho) [reply mods/ra/sequences/aircraft.yaml]: There's nothing to be reverted as the mh60 sequnce is right behind it. I will delete the unnecessary tabs

**Comment by Punsho** [comment]
> How does this affects the campaign? Repurposing the Hind to the transport from it's attack copter role might affect mission performance. Note that in legacy RA1 the Chinook was shared between both factions, so this pretty much shifts the Hind to take over both helis' place in missions.

I haven't played all missions myself, I started this pr to get more feedback and people involved as well for having documented discussion.

> I personally think we shouldn't transform the hind in a transport helicopter. One reason is that the sprite is so thin and small, it just does not look right to me. The other point is that we can not give soviets air transport as a passive ability of an already powerful weapon. You should have to invest into (mass) air transport. If soviets really need a transport heli, then let's give them the Chinook, but leave the hind in original state. We diverge even further from it with the proposed setup.

The hind is thin so he can carry just 4 passengers, we could give both faction the Chinook but then the Hind sprite goes unused and the Hind / Chinook dynamic is lost: Chinook would be faster so it can get to a place quicker, have a possibility unload rocket soldiers to ward off other aircraft, and then the Hind is slower but it has a builtin airborne weapon. Also 4 slot for soviets is really good as they will be mainly transporting low amounts of infantry anyway: 4 flamers, 4 grenadiers, 2 thiefs, 1 engineer and so on

> From a balance/gameplay perspective I think this needs more testing. I haven't seen too much wide scale testing, and I think these kind of changes need to be thoroughly vetted.
> 
> The few games I did test I didn't see the unit used, except my for my few failed attempts to abuse it.

I agree, more testing is necessary. I wanted to start this pr before that was done. From what I seen in games where Hind was used in, the gun was almost never useful in direct combat engagements. People always failed to abuse the infinite airborne weapon. It's doing exactly what I created it to do, nothing else.

> Also, I recently realized it's a bit strange that it fires so slowly. I get this is how the Blackhawk works in RA2, but it still doesn't make sense. A fast rate of fire weapon with low damage per hit would be better.

Well my first idea was to make sounds between Hind and BlackHawk distinct, not sure if this is the best solution though.

**Comment by GraionDilach** [comment]
Come to think of it - the aim of this PR could be split to two PRs, limiting this PR to add in the new Allied copter art and disable the Hind to be built alone and a followup readjusting the Hind into the new role. Such a method would give more time for the community to decide upon the Hind while missions relying on it would remain intact.

**Comment by pchote** [comment]
That was our original plan, which was preempted by this PR.

**Comment by Inq8** [comment]
Here is an alternate mh60 that was squashed & re-rendered to be similar dimesions to the original hind. how do you feel about this one? 
fatshaming

- Thinner
- Black landing gear
- Slightly smaller

[mh60alternate.zip](https://github.com/OpenRA/OpenRA/files/3236420/mh60alternate.zip)

**Comment by GraionDilach** [comment]
> 
> That was our original plan, which was preempted by this PR.

Stick with it then. Considering that there were discussions about playtest preparations, I'd say aim to finish/prioritize this art replacement to Next and delay the reintroduction of Hind to Next+1 even. Sounds like a win-win for everyone - a cycle would give more than enough time for the new-Hind discussions to form up and reach a point, while the artwork replacement would already give a better light to the crowd which considered the gameplay reasoning behind the current state an excuse.

**Comment by Punsho** [comment]
Alright, I'll split the pr if it is wanted

**Comment by matjaeck** [comment]
@Inq8 That looks better from my perspective. 

I agree that we should defer this and leave it to someone who is able to address requests. @Punsho please leave this to someone else. Your comment did not address any of the concerns that I raised and you just ignored everything, stating it would be fine as it is. We do not have the resources to make every PR of you a personal lesson for you. I don't trust you with this and think you are not experienced enough to handle this. You haven't made the artwork, you are not able to convince others of the idea behind this PR and you put very very low effort in this PR. I don't trust you with this and I am convinced that it will hurt the quality of the result if you author this PR.

This topic is too important to make it a contributing tutorial. Resolute :-1: from me unless this is handled by somebody experienced.

**Comment by matjaeck** [comment]
Updated my last comment, since it was written quite emotionally and thus not too helpful.

**Comment by Punsho** [comment]
Replaced Black Hawk cameo with the Inq's fixed version

**Comment by GraionDilach** [comment]
@Punsho I believe from the sheer amount of thumbsups on my previous comment that a consensus have been formed upon the scope of this request. Please remove all yaml edits you made on the Hind and give it a `~disabled` prerequsite.

**Comment by Inq8** [comment]
> Hind irl is a armoured gunship that can carry up to 8 infantry. UH-60 Black Hawk is more of a utility helicopter with a maximum cary capacity of 20 infantry. One of the solutions was to make Black Hawk thinner but people seem like the "thicc" version more. The thickness also helps to visually differentiate it from the other helicopters. Just a fact dump for context and future discussions.

The helicopter modelled is actually a MH-60L DAP, its not a UH-60, its a gunship used by the SOAR unit in the US Army. It's the closest thing to a Hind in the US / NATO arsenal. While a normal "Blackhawk" (MH-60L) can 20 guys, the DAP (Direct Action Penetrator) sacrafices that carrying capacity to put as many guns, missiles & rockets on the airframe as possible.

Its loadout varies on its mission, but typically it has a 30mm cannon, x2 Miniguns & a rocket pod. It is also possible for it to carry Hellfire missiles & Stringer AA missiles.

http://www.airwar.ru/image/idop/ah/ah60/ah60-5.jpg

DAP

DAP2

**Comment by Punsho** [comment]
This pr has been split into #16618 as requested.

File: mods/ra/rules/aircraft.yaml
Comment (pchote) [inline mods/ra/rules/aircraft.yaml]: Missing comma after `~disabled`

File: mods/ra/rules/aircraft.yaml
Comment (pchote) [inline mods/ra/rules/aircraft.yaml]: Please either call this `mh60` or rename the sequence and sprite to `blck`. There is no need for these to be inconsistent.

**Review by pchote — COMMENTED** [review]
Testing ingame, I think we should switch to the alternate sprite posted above. Not because it looks better (I still prefer the original), but because its size is more in line with the longbow and hind. If we take the "chubby" version and also move ahead with #16618 I just see the arguments about how the new version should be the transport instead.

The cameo is noticeably blurry compared to the other aircraft. Please run a sharpen filter on it after resizing but before adding the text.

Otherwise looks generally good. This topic has been debated at length for years now, and i'm happy to move ahead with this solution.

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: fixed

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: for now I named it to mh60

**Comment by Punsho** [comment]
Welp, as Inq just commented it is a utility helicopter turned gunship. It is fat as the original helicopter was also fat and this is the explanation we could run with. For now I updated it as Inq has added more details but I don't think we're finished talking chubbyness

File: mods/ra/rules/aircraft.yaml
Comment (pchote) [inline mods/ra/rules/aircraft.yaml]: Please keep this in caps to match the other unit definitions.

File: mods/ra/rules/aircraft.yaml
Comment (pchote) [inline mods/ra/rules/aircraft.yaml]: `MH60.Husk` and belongs in `husks.yaml` to match the other aircraft husks.

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: done

File: mods/ra/rules/aircraft.yaml
Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: moved it

File: mods/ra/rules/husks.yaml
Comment (abcdefg30) [inline mods/ra/rules/husks.yaml]: Please revert this change.

Comment (Punsho) [reply mods/ra/rules/husks.yaml]: nothing to be reverted, deleting tabs

File: mods/ra/rules/husks.yaml
Comment (abcdefg30) [inline mods/ra/rules/husks.yaml]: This now needs to be `MH60`, too! (See travis error logs.)

**Review by abcdefg30 — DISMISSED** [review]

File: mods/ra/rules/husks.yaml
Comment (abcdefg30) [reply mods/ra/rules/husks.yaml]: Actually, now that I think of it: It is probably better to change `MH60` back to `mh60` in the sequence yamls. Apparently we have all names upper case in the rules, but lower case in the sequences.

Comment (Punsho) [reply mods/ra/rules/husks.yaml]: I see, made it in sequences be lower case

**Comment by Inq8** [comment]
Ran a sharpness filter over the icon, hows that?
[mh60icon.zip](https://github.com/OpenRA/OpenRA/files/3246088/mh60icon.zip)
mh60icon0000 Old
mh60icon20000 New

**Comment by pchote** [comment]
Yes, that looks better.

@Punsho can you please add the updated icon and rebase this on the latest bleed? It would be useful to have the `Hovers` fix so that we can properly check the firing offsets.

**Comment by Punsho** [comment]
Rebased. Icon updated

**Review by pchote — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]
I do like @Inq8's model, but remain skeptical of the general plan behind this. We should move on nevertheless and I hope you guys are more convinced that this is the right thing to do than I am.

**Comment by matjaeck** [comment]
I have come to the conclusion that we shouldn't do this, and instead should keep everything as it is. The new unit sets precedence to add more custom units and will not help to please people who have complained about the the lack of authenticity in the past. Removing the hind is also not the solution. Since all alternatives worsen the situation for people who value authenticity, and additionally challenge loyalty of players and contributors, it's not worth it. The chance that we really improve OpenRA's image wrt authenticity is very low and the risk that people who like OpenRA lose interest seems high. I always wanted the hind to be removed from allies but after all the discussions I think it is best to leave it untouched. So :-1:, as this is my honest opinion.

**Comment by FiveAces** [comment]
My perspective is that of a competitive player rather than a developer, so take my opinion here with a grain of salt. The HIND issue has been a sore spot for the OpenRA project ever since I started playing four years ago, and multiple solutions were proposed. All of them failed to gain traction, and this particular one is probably no exception, but I still think it's the best solution to date.

Before giving a final verdict, I want to discuss the most commonly proposed ideas and list their pros and cons:

**-Restoring the HIND as a Soviet unit without compensating Allies:**
This is the minimum effort solution; all it requires on a technical level is changing around five-ish lines of code. So, why would that be a bad solution? The answer is, it probably wouldn't be for non-competitive players or singleplayer-only users. The multiplayer, however, arguably a focal point of the OpenRA project, would suffer tremendously because the HIND right now serves a crucial role in the arsenal of Allies by hardcountering V2s and softcountering infantry. Were it to be moved over to Soviets, there would be a big gap left in the rock-paper-scissors dynamics that OpenRA relies on: Soviets would rise in power level so dramatically upon reaching radar tech that there would be no point in continuing a match as Allies after an opponent reaches that stage of the game. This is objectively undesireable, and that solution should be discarded accordingly.

**Restoring the HIND to Soviets while giving the Allies Longbows on Radar tech level:**
While this is close to how vanilla Red Alert handled the tech tree, this comes with its own set of implications. First off, it still deprives Allies of their most valuable anti-infantry tool (in competitive matches, Artilleries are an easy target for mobile units such as the Mobile Flak or the Yak, and thus can't reliably fulfill that role); moreover, Longbows are balanced around being high-tech unit that destroy heavily armored targets such as Harvesters and Heavy Tanks, which effectively means that this change would take the frail house of cards that is multiplayer balance and throw it into a tornado just to see where the cards will land. It would also further skew unit diversity in favour of Soviets, depriving Allies of a gunship archetype and leaving Soviets with two redundant units that fulfill very similar roles.

**Developing a new gunship for Allies and giving HINDs over to Soviets as a transport unit:**
My preferred solution, albeit with its own set of problems. First and foremost, there's no other gunship in vanilla RA, so oldschool players might feel alienated. The HIND is a very slim helicopter, so it having the capability to transport an entire army might feel out of place. My personal take on the situation would be to turn it into a Carryall instead, able to lift a single vehicle into the air, which would match its slim model more closely given there's rendering technology to visibly attach the vehicle to the helicopter while airborne. 

To sum it up, I don't think the current proposal is an amazing one or without flaws, but I genuinely think it is the _least bad_ one, so I give it a hesitant thumbs up.

**Review by abcdefg30 — DISMISSED** [review]
Something is wrong with attack behaviour. Testcase: Force fire on the ground.

**Comment by reaperrr** [comment]
> Something is wrong with attack behaviour. Testcase: Force fire on the ground.

I didn't notice anything odd (and yes, I compared to the Hind to be sure), and out of curiosity I compared their rules, too: except for name and buildability they're 100% identical, so I don't see how it could be this PR's fault.

> The new unit sets precedence to add more custom units

We already have the soviet Flak Truck, as well as a completely new sprite for the Chinook (and APC turret in TD).
IMO considering how far our RA/TD mods already stray from the originals in terms of gameplay, I see no reason to reject one more new unit out of traditionalism.
In fact, personally I find the APC (M113) on Soviets nearly as offending as the Hind on Allies and would rather replace it with the BTR-based APC sprite from my TDX mod, but let's leave that discussion for another day/PR.

In my opinion we should take this PR as-is and see what ~flak~ feedback we get from the first playtest/newspost. We can then still revert it if the majority considers it no better or worse than the Hind.

If the overall feedback basically says "Removing Hind from Allies is good, but that custom unit/graphics is bad", the only other idea I can come up with would be to give Allies two different Apaches: add normal Apache (TD-style) with more or less Hind specs, and give Longbow slightly customised gfx (radar dome on rotor, rocket pods under wings). I might be able to whip up that semi-custom Longbow if that idea gains any traction.

About what to do with the Hind, if nobody has a fully convincing idea about in what capacity to give the Hind back to Soviets, we should just leave it disabled in MP for the upcoming release and try to come up with something thought-through for Next+1 (which will hopefully be a shorter release cycle).

**Comment by abcdefg30** [comment]
Sorry, I don't know how you can miss this. At this point I am really not sure if you actually tested the PR, are trolling me or if this is some weird issue on my end (which I somehow doubt, since I tested this 3 times now, and on another machine and on a new fork). I'll give you the steps and some gifs so you can verify this bug yourself: First fetch the PR, then rebase it onto bleed (which should always be done when testing PRs) and rebuild (which should be done after rebasing in case code changes were introduced via the rebase or the PR). Then just start any RA map and attack anything. Example for Hind vs Hawk behaviour for force firing on the ground:
firinghind
firinghawk

**Comment by GraionDilach** [comment]
`AttackAircraft>AttackType: Hover` is missing - it's indeed the lack of rebases. The PR'S YAML wasn't updated with the recent aircraft refactorings.

**Comment by reaperrr** [comment]
>  then rebase it onto bleed

OK, I didn't. My bad.

**Comment by Punsho** [comment]
Rebased and `AttackType: Hover` added

File: mods/ra/rules/aircraft.yaml
Comment (abcdefg30) [inline mods/ra/rules/aircraft.yaml]: Please add a new line here.

Comment (Punsho) [reply mods/ra/rules/aircraft.yaml]: Added

**Comment by abcdefg30** [comment]
Imho we should let the AI build Hawks instead of Hinds too.

**Comment by Punsho** [comment]
replaced all hind references in ai.yaml with mh60

**Comment by Smittytron** [comment]
I'd just have mh60 inherit hind and not give hind build prereqs. I want to leave hind in the game for future campaign missions that have soviet hinds.

**Comment by abcdefg30** [comment]
That won't work together with #16618.

File: mods/ra/rules/ai.yaml
Comment (reaperrr) [inline mods/ra/rules/ai.yaml]: Missing `:`

Comment (Punsho) [reply mods/ra/rules/ai.yaml]: fixed

**Comment by Inq8** [comment]
Whilst I contributed the artwork to this PR, I have refrained from commenting for or against, until now.

 I do believe @Reaperrr summarises it best:

Some players are against new artwork, full stop. (Classic Players)

Some players are against additional new artwork. However, those same people currently tolerate the 3rd party chinook, Flak truck & AA APC sprites. (Hypocrites)

Others are indifferent.

In addition to above, people  are against the Allied hind - this comprises of two groups:

Players that are against changes to the original unit trees ( Classic Players) That flat out reject any changes made no matter what. 

Players that are against a historically  iconic Soviet aircraft being fielded by France & England. (Lore Players)

Finally the people for the Allied hind:
Players that disregard the previous two arguments in favour of gameplay. (Gameplay Players)

Taking all groups into account, I firmly believe that a re-skin is the best way forward.

As it avoids changes in gameplay (doesn’t aggravate gameplay players) & also avoids criticism from lore players.

It will upset hypocrites & classic players... however you will always be criticised by “Classic Players” no matter what gameplay or artwork changes are made. Even if the Apache was added rather than the Blackhawk.

So just reiterating what @Reaperrr said, 
I fully support this PR going ahead into a playtest to see what public criticism is made & it can be adjusted from there. 

(Whether that is cancelling it for release, or replacing it with the Apache)

**Review by pchote — APPROVED** [review]

**Comment by matjaeck** [comment]
> Some players are against additional new artwork. However, those same people currently tolerate the 3rd party chinook, Flak truck & AA APC sprites. (Hypocrites)

Your mixing quite a few things here: 1) this is a RA PR and you come up with the TD APC. 2) the chinook is not a completely new unit but an visual enhanced existing unit that would iirc otherwise be used in two mods. 3) the paradigm you outline here leaves people two options: uninstall the game, leave the community, delete all their stuff and never come back or be a stay involved but be a hypocrite. The nowadays active OpenRA community doesn't give a shit about old RA - what else than tolerating things like the flak truck shoud the "classic" people do in your opinion? Shut up or leave, right??

@pchote you can pretend and maybe even believe "asset thief is not going to happen under your watch" but reality doesn't care. The subject of this PR is not a fundamental problem, but the attitude and paradigm here and in other issues and PRs is. It will be the death of the project as you know it, and the stolen assets are going to be everything what will be left of RA. So be it.

**Review by reaperrr — APPROVED** [review]
I don't think we'll make any more progress purely by discussion among a few people, so LGTM if only to get a larger sample of feedback through the upcoming playtest

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ed32c88459f4928a0ec34191a61397b1592487ed...fcecc67602ad93157b9138d71552258d67600cd9

**Comment by Jehoel** [comment]
If it's not too late - another option is to give the Hinds back to the Soviets (where's it's almost redundant with the Yak, I know) and let the Allies get the Longbow without the Tech Center, but the Longbow won't have Hellfire rockets until the Tech Center is built (and the Longbow always uses guns against infantry, but Hellfire rockets against vehicles and structures).

...or rebrand it as the Huey (1956) which is more period-correct than the Blackhawk (1974+) if you're in the same lore-school as me who maintain OpenRA takes place in the 1950s.

**Comment by matjaeck** [comment]
> what else than tolerating things like the flak truck shoud the "classic" people do in your opinion? Shut up or leave?

@Inq8 that was a serious question btw and your reply is still welcomed since I still haven't found an answer for myself how to deal with the hypocrisy blame / participate without being a hypocrite.

**Comment by Inq8** [comment]
Hey @matjaeck, my comment was a general statement & a personal opinion on why it is so difficult to please every party on a PR like this. 
It wasn’t intended as flame-bait.

I personally like to play the game in a more classic style without the flak trucks & with Soviet hinds, but with benefits of the RA2 style UI. Therefore I enjoy playing OpenRA more than the original.

At the same time, I understand there is a competitive multiplayer scene within OpenRA  - and in this reimagining of RA it has new units to fill holes in the questionable/poorly balanced factions Westwood version of the game.

I accept that my imagining of how the game should play is not a popular opinion & therefore don’t force anyone to play it that way, however with the flexibility of OpenRA I can still play the way I want to with mods & mod maps etc.

Therefore to answer your rather loaded question.
I would say either;

1. Play mod-maps or mods to make the game play how you want.

2. If you think your opinion is shared with a large portion of the player base, Garner enough vocal support to get the game changed to play the way you want.

**Comment by GraionDilach** [comment]
Come to think of it, the RA Classic mod could be offered as an official bonus download somewhere to give it a PR stunt, since it's already within the OpenRA account and semi-completed for the most part.

**Comment by matjaeck** [comment]
@Inq8 Thanks for taking the time to answer. I'll accept these things and look around for something new that is compatible with my interests and motivations.

**Comment by matjaeck** [comment]
https://github.com/electronicarts/CnC_Remastered_Collection/blob/ae72fce5ddd6a12bf48d00e0db8a400b821c2529/REDALERT/DEFINES.H#L1716

**Comment by deleted-user-1** [comment]
https://github.com/electronicarts/CnC_Remastered_Collection/blob/ae72fce5ddd6a12bf48d00e0db8a400b821c2529/REDALERT/DEFINES.H#L1716

`Soviet attach helicopter.`

220px-Kaman_K-1200_K-Max_2011_01

**Pull request merged**
